scripts/cleanup_dp.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Configure the AWS & kubectl environment
+
+mkdir ~/.aws; touch ~/.aws/config
+
+export AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY
+
+export AWS_REGION=ap-south-1
+export AWS_DEFAULT_OUTPUT=json
+
+export region=ap-south-1
+export cluster_name=uat-cluster
+
+echo "Region: $region"
+echo "Cluster name: $cluster_name"
+echo "Pull Request Number: $PULL_REQUEST_NUMBER"
+
+sts_output=$(aws sts assume-role --role-arn "$AWS_ROLE_ARN" --role-session-name dp-session-script)
+
+export AWS_ACCESS_KEY_ID="$(echo "$sts_output" | jq -r '.Credentials.AccessKeyId')"
+export AWS_SECRET_ACCESS_KEY="$(echo "$sts_output" | jq -r '.Credentials.SecretAccessKey')"
+export AWS_SESSION_TOKEN="$(echo "$sts_output" | jq -r '.Credentials.SessionToken')"
+
+aws eks update-kubeconfig --name "$cluster_name" --role-arn "$AWS_ROLE_ARN"
+
+### Get list of helm charts
+deployed_charts="$(helm ls -A | awk '/ce/ {print $2}')"
```

Comment:
I just ran this `helm ls` command and there's a lot of columns and it felt fragile. Should we just go with JSON here as well?

```suggestion
deployed_charts="$(helm ls -A --filter 'ce[0-9]+' --no-headers --output json | jq -r '.[].namespace')"
```

This feels more stable, and also states explicitly that we're after the _namespaces_ for each installation.